app/concepts/lockstep/client.rb

```diff
@@ -159,8 +159,8 @@ def put(path, body: {}, params: {})
       request(:put, path, body, params)
     end
 
-    def delete(path)
-      request(:delete, path, {}, {})
+    def delete(path, body: {})
```

Comment:
is this change impacting the older places where we are using this?

Reply:
No this would not, since body has a default value. I checked the normal destroy which calls this method. THe evidence for the same is on the PR